### ISSUE TITLE
PyPI tarball contains code with syntax errors

### ISSUE THREAD

**Issue by adamjstewart** [alvations/sacremoses#91]
The latest release on PyPI (0.0.38) contains code with syntax errors. In the `sacremoses/sent_tokenize.py` file, I see:
```python
...
        for i, token in enumerate(iter(tokens[:-1])):                           
            if re.search(IS_EOS, token)     
```
During installation, this leads to the following warning message:
```
running install_scripts
Installing sacremoses script to /Users/Adam/spack/opt/spack/darwin-catalina-x86_64/clang-11.0.3-apple/py-sacremoses-0.0.38-jrvwvizhy453s2pooq6udlw3b7my5v4e/bin
  File "Users/Adam/spack/opt/spack/darwin-catalina-x86_64/clang-11.0.3-apple/py-sacremoses-0.0.38-jrvwvizhy453s2pooq6udlw3b7my5v4e/lib/python3.7/site-packages/sacremoses/sent_tokenize.py", line 69
    if re.search(IS_EOS, token)
                              ^
SyntaxError: invalid syntax
```
I noticed that this file doesn't even exist in the GitHub repo. I also noticed that the GitHub repo does not have this 0.0.38 release. What's going on here?

**Comment by alvations** [comment]
Thank you for raising the issue. An unfinished code leaked out in the previous package. We're updating the package by end of today.

**Comment by alvations** [comment]
Patched with v0.0.39. Sorry again for the inconvenience. 

```
pip install -U sacremoses==0.0.39
```

**Comment by adamjstewart** [comment]
New tarball looks good, thanks for the quick fix!
